python/tests/ingestion/test_graphql_ahr_measure_ids.py

```diff
@@ -0,0 +1,70 @@
+import json
+import os
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+BASE_DIR = os.path.join(THIS_DIR, os.pardir, os.pardir, 'ingestion')
+CONFIG_FILE_PATH = os.path.join(BASE_DIR, 'graphql_ahr_measure_ids.json')
+
+with open(CONFIG_FILE_PATH, 'r') as file:
+    data = json.load(file)
+
+
+def get_measure_ids(demographic: str, measure_name: str, flatten=False):
+    """
+    Retrieve measure IDs based on demographic and measure name.
+
+    Args:
+    demographic (str): One of 'all', 'age', 'race_and_ethnicity', 'sex'.
+    measure_name (str): The name of the measure (e.g., 'Asthma', 'Diabetes').
+    flatten (bool): Whether to flatten the list of IDs if the result is a dictionary.
+
+    Returns:
+    list: A list of measure IDs.
+    """
+    if demographic in data:
+        for measure in data[demographic]:
+            if measure['measure'] == measure_name:
```

Comment:
this might be an inefficient way to do this; you're just looking for the single `measure_name` right? so instead of iterating over all of them until one matches, you should just be able to use something like `data[demographic].get(measure_name)` or similar?